ios/src/TGLabelPickResult.h

```diff
@@ -0,0 +1,24 @@
+//
+//  TGLabelPickResult.h
+//  tangram
+//
+//  Created by Karim Naaji on 11/02/16.
+//
+
+#import <Foundation/Foundation.h>
+#import "TGGeoPoint.h"
+
+typedef NS_ENUM(NSInteger, TGLabelType) {
+    TGLabelTypeIcon = 0,
+    TGLabelTypeText,
+};
+
+@interface TGLabelPickResult : NSObject
+
+- (id) initWithCoordinates:(TGGeoPoint)coordinates type:(TGLabelType)type properties:(NSDictionary *)properties;
```

Comment:
Same argument here for `nullable`, if this happens to not be the default, we can start to explicitly add them.

Good call on `instancetype, I will update all the other initializers.